cmd/lakefs/cmd/run.go

```diff
@@ -53,6 +53,9 @@ var runCmd = &cobra.Command{
 		// validate service names and turn on the right flags
 		dbParams := cfg.GetDatabaseParams()
 
+		if err := db.ValidateSchemaUpToDate(conf.GetDatabaseParams()); err != nil {
+			logger.WithError(err).Error("Failed on schema validation")
```

Comment:
Need to exit or use Fatal

Reply:
Considered doing that,
But then no one could run a previous version even if they wanted.